We gave up this exploit. For more info see exploit.py.
